huaweicloud/provider_test.go

```diff
@@ -335,3 +339,15 @@ func envVarFile(varName string) (string, error) {
 	}
 	return tmpFile.Name(), nil
 }
+
+func testAccPreCheckScm(t *testing.T) {
+	if HW_CERTIFICATE_KEY_PATH == "" {
+		t.Skip("HW_CERTIFICATE_KEY_PATH must be set for SCM certificate tests")
+	}
+	if HW_CERTIFICATE_CHAIN_PATH == "" {
+		t.Skip("HW_CERTIFICATE_CHAIN_PATH must be set for BMS certificate tests")
+	}
+	if HW_CERTIFICATE_PRIVATE_KEY_PATH == "" {
+		t.Skip("HW_CERTIFICATE_PRIVATE_KEY_PATH must be set for BMS certificate tests")
+	}
```

Comment:
can be combined to one if, if HW_CERTIFICATE_PATH == "" || if HW...

Reply:
fixed